unit-tests/src/org/commcare/android/shadows/SQLiteDatabaseNative.java

```diff
@@ -0,0 +1,305 @@
+package org.commcare.android.shadows;
+
+import java.util.Locale;
+import java.util.Map;
+
+import net.sqlcipher.SQLException;
+import net.sqlcipher.database.SQLiteDatabase.CursorFactory;
+import net.sqlcipher.database.SQLiteDatabaseHook;
+import net.sqlcipher.database.SQLiteException;
+import net.sqlcipher.database.SQLiteStatement;
+
+import org.robolectric.annotation.Implements;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+
+/**
+ * @author ctsims
+ *
+ */
+@Implements(net.sqlcipher.database.SQLiteDatabase.class)
+public class SQLiteDatabaseNative {
+    private android.database.sqlite.SQLiteDatabase db;
+    
+    public void __constructor__(String path, char[] password, CursorFactory factory, int flags) {
+        db = android.database.sqlite.SQLiteDatabase.openDatabase(path, null, flags);
+    }
+    
+    public void __constructor__(String path, char[] password, CursorFactory factory, int flags, SQLiteDatabaseHook hook) {
+        db = android.database.sqlite.SQLiteDatabase.openDatabase(path, null, flags);
```

Comment:
Yeah, it's a potential issue to check for in the future, but the incoming Hook here is the sqlcipher type, so we can't forward it. We'd have to make another wrapper like we did for the cursor, which is a super pain due to the way these SQL classes do a bunch of shit in their constructors.